sdl2-vga-terminal/src/VgaTerminal.hpp

```diff
@@ -5,36 +5,42 @@
 #include <string>
 #include <bitset>
 
-
 class VgaTerminal : public Window
 {
-
 public:
     typedef struct
     {
-        uint8_t  mode;    // video mode (only mode 3 available at the moment)
-        uint8_t  tw;      // terminal width
-        uint8_t  th;      //          hieght
-        uint8_t  cw;      // char     width
-        uint8_t  ch;      //          height  | font size
+        uint8_t  mode;      // video mode (only mode 3 available at the moment)
+        uint8_t  tw;        // terminal width
+        uint8_t  th;        //          height
+        uint8_t  cw;        // char     width
+        uint8_t  ch;        //          height  | font size
+        uint16_t numColors; // 65K max (16 bit) otherwse 24|32 bits if 0 (?)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [struct member 'videoMode_t::numColors' is never used.](https://app.codacy.com/manual/Raffaello/sdl2-vga-terminal/pullRequest?prid=5403184)